.travis.yml

```diff
@@ -18,16 +18,17 @@ services:
 install:
 - docker pull ${LEARNOCAML_SERVER_IMAGE}
 - docker pull ${LEARNOCAML_CLIENT_IMAGE} 
-- docker run -d -i --name=LEARNOCAML_SERVER -v learn-ocaml-sync-demo:/sync -v $PWD/pfitaxel-server/src:/repository:ro ${LEARNOCAML_SERVER_IMAGE}
-- docker run -d -i --link LEARNOCAML_SERVER:LEARNOCAML_SERVER --name=LEARNOCAML_CLIENT -v ${TRAVIS_BUILD_DIR}/tests.sh:/tests.sh ${LEARNOCAML_CLIENT_IMAGE}
+- docker network create learnocaml
+- docker run -d --net=learnocaml --name=LEARNOCAML_SERVER -v $PWD/test-directory/pfitaxel-server/src:/repository:ro ${LEARNOCAML_SERVER_IMAGE}
+- docker run -d -i --net=learnocaml --name=LEARNOCAML_CLIENT -v ${TRAVIS_BUILD_DIR}/test-directory/tests.sh:/tests.sh -v ${TRAVIS_BUILD_DIR}/learn-ocaml.el:/learn-ocaml.el ${LEARNOCAML_CLIENT_IMAGE}
 - docker ps -a
-- chmod a+x ${TRAVIS_BUILD_DIR}/tests.sh
 
 script:
-- echo -e "${ANSI_YELLOW}Executing tests... ${ANSI_RESET}" && echo -en 'travis_fold:start:script\\r' 
-- docker exec -it LEARNOCAML_CLIENT sh /tests.sh
-- docker exec -it LEARNOCAML_SERVER touch demo.ml
-- docker exec -it LEARNOCAML_SERVER learn-ocaml-client --id=demo --server=http://localhost:8080 --token=ZCB-3LZ-AC4-3O4 demo.ml 
+- docker exec -it LEARNOCAML_CLIENT sudo chmod +x /tests.sh 
```

Comment:
@leunam217 here, same remark as in https://github.com/pfitaxel/learn-ocaml.el/pull/4#issuecomment-499551821